packages/client/src/consts.js

```diff
@@ -48,6 +48,8 @@ export const PLAN_REJECTED = 'rejected'
 export const CHANNEL_RFI_STATUS = 'rfis'
 export const CHANNEL_MAPPING = 'mapping'
 
+export const NATURAL = 'Natural'
```

Comment:
@sanjayradadiya - I don't like us declaring these constant values twice.  Is it possible to just use the constants declared in `@serge/config` below?